addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/XxeScanRule.java

```diff
@@ -405,4 +422,43 @@ static String createLfrPayload(String requestBody) {
         sb.append(requestBody.substring(endIdx));
         return sb.toString();
     }
+
+    private boolean localFileReflectionRaiseAlert(
+            HttpMessage msg, String requestBody, String payload) throws IOException {
+        for (int idx = 0; idx < LOCAL_FILE_TARGETS.length; idx++) {
```

Comment:
Should have a `isStop()` check inside the loop, it can also return true, in which case the other `isStop` checks above are no longer needed (e.g. `alertRaised` → `stop`).